files/class-vip-filesystem.php

```diff
@@ -494,16 +495,20 @@ public function update_attachment_meta() {
 			return;
 		}
 
-		$updater = new Meta_Updater( 1000 );
+		$batch_size = 3000;
```

Comment:
I notice that this change doesn't include the batch size modifications for specific sites

Reply:
Yeah, was trying to avoid referencing clients directly since this is a public repo.

Reply:
AH that makes sense